demo-app/src/app.rs

```diff
@@ -91,93 +85,77 @@ impl StateTransitionRunner<ZkConfig> for DemoAppRunner<ZkDefaultContext> {
 }
 
 #[cfg(feature = "native")]
-pub fn create_mock_context_genesis_config(
-    sequencer_address: Address,
-    sequencer_da_address: Vec<u8>,
+pub fn create_config(
+    initial_sequencer_balance: u64,
+    value_setter_admin_private_key: &DefaultPrivateKey,
+    election_admin_private_key: &DefaultPrivateKey,
 ) -> GenesisConfig<DefaultContext> {
-    create_demo_genesis_config::<DefaultContext>(sequencer_address, sequencer_da_address)
+    create_genesis_config::<DefaultContext>(
+        initial_sequencer_balance,
+        generate_address::<DefaultContext>(SEQ_PUB_KEY_STR),
+        SEQUENCER_DA_ADDRESS.to_vec(),
+        value_setter_admin_private_key,
+        election_admin_private_key,
+    )
 }
 
-pub fn create_demo_genesis_config<C: Context>(
+#[cfg(feature = "native")]
+fn create_genesis_config<C: Context>(
+    initial_sequencer_balance: u64,
     sequencer_address: C::Address,
     sequencer_da_address: Vec<u8>,
+    value_setter_admin_private_key: &DefaultPrivateKey,
```

Comment:
Why it is called `value_setter_admin_private_key` and not just `admin_private_key`? When I see `setter` I expect some kind of callable, like Fn

Reply:
Ok, I see. Can you please add comment on why it is called like that then?